src/components/feedback/components/forms.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ControlTextarea from '@mapbox/mr-ui/control-textarea';
+import classnames from 'classnames';
+
+// character limit for the feedback textarea
+export const feedbackLimit = 1000;
+export const feedbackMinimum = 3;
+
+// Textarea for the user to submit text feedback
+export class FeedbackTextarea extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feedback: ''
+    };
+    this.handleFeedback = this.handleFeedback.bind(this);
+  }
+
+  handleFeedback(value) {
+    this.setState({ feedback: value }, () => {
+      this.props.onChange({ value, overLimit: this.isOverLimit() });
+    });
+  }
+
+  isOverLimit() {
+    return this.state.feedback.length > feedbackLimit;
+  }
+
+  render() {
+    const { id, label, placeholder, value, validationErrorMinimum } =
+      this.props;
+    const feedbackLength = this.state.feedback
+      ? feedbackLimit - this.state.feedback.length
+      : feedbackLimit;
+    return (
+      <>
+        <div className="relative">
+          <ControlTextarea
+            themeControlTextarea="textarea hmin120 bg-white"
+            themeLabel="txt-m mb6"
+            id={id}
+            label={label}
+            value={value}
+            onChange={this.handleFeedback}
+            placeholder={placeholder}
+            validationError={
+              (validationErrorMinimum ? 'Tell us more' : '') ||
```

Comment:
@HeyStenson Do you have any suggestions for the error message that will appear when the user's feedback does not meet the minimum? I'm not sure if "Tell us more" is helpful here or needs a little more instruction 🤔

Reply:
Hmmm, been thinking about this for a lil while and I really can't think of anything better 😅  The only punchup I can think of that might make a difference is "Tell us more!" to make it seem more upbeat. (Other option would be "We need more detail" but I don't think that strikes the right tone.)